src/accelerate/utils/dataclasses.py

```diff
@@ -616,18 +613,21 @@ def save_model(self, accelerator, model, output_dir, model_index=0):
         from torch.distributed.fsdp.fully_sharded_data_parallel import FullyShardedDataParallel as FSDP
         from torch.distributed.fsdp.fully_sharded_data_parallel import StateDictType
 
-        if self.state_dict_type == StateDictType.FULL_STATE_DICT:
+        if is_torch_version("<=", "1.13.0"):
```

Comment:
```suggestion
        if is_torch_version("<=", "1.13.5"):
```
Just in case they make a patch release before 1.14 is out.